scripts/installer/install

```diff
@@ -5,6 +5,7 @@ use warnings;
 use Getopt::Long;
 use Socket;
```

Comment:
I'd lean towards removing timing-only changes from the stock install script; but that's an open question.

Reply:
I removed them and the changes reflect in the latest commit